runtime/manta/src/lib.rs

```diff
@@ -410,6 +410,10 @@ parameter_types! {
     pub DrawingFreezeout: BlockNumber = prod_or_fast!(1 * DAYS, 1 * MINUTES);
     /// Time in blocks until a collator is done unstaking
     pub UnstakeLockTime: BlockNumber = LeaveDelayRounds::get() * DefaultBlocksPerRound::get();
+    /// JumboShrimp CurrencyId
+    pub JumboFarmingCurrencyID: MantaAssetId = 8;
+    /// Farming PoolId for JUMBO token
+    pub JumboShrimpPoolId: PoolId = 0;
```

Comment:
I thought we can add `setJumboshrimpPoolId` and `setJumboFarmingCurrencyID` into lottery module. if we're using vManta instead of manta, we don't know the assetId yet. the same as pool id.

Reply:
or add these two setting into `set_farming_params`?

Reply:
done